src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -246,6 +246,16 @@ public async Task<IActionResult> OnPostToggleApprovalAsync(string id, string rev
             await _manager.ToggleApprovalAsync(User, id, revisionId);
             return RedirectToPage(new { id = id });
         }
+
+        public IActionResult OnGetUpdatePageSettings(bool hideLineNumbers = false, bool hideLeftNavigation = false)
+        {
+            UserPreferenceModel userPreference = _preferenceCache.GetUserPreferences(User.GetGitHubLogin());
```

Comment:
We should handle scenario when user cache is missing. for e.g. cache got timed out and then user clicked on hide line number. This will throw null pointer in that case.

Reply:
I fixed this with automapper